XAMLator.Server.Shared/MonoEvaluator.cs

```diff
@@ -6,110 +6,168 @@
 
 namespace XAMLator.Server
 {
-	public class Evaluator : IEvaluator
```

Comment:
There are indentation changes (tabs to spaces) that makes it impossible to review. Can you fix the indentation please?